modals.css

```diff
@@ -0,0 +1,175 @@
+:root {
+	--primary-misc-color: #bbb;
+	--primary-btn-hover-color: #ccc;
+	--primary-btn-active-color: #aaa;
+
+	--secondary-misc-color: rgb(26, 115, 232);
+	--secondary-btn-hover-color: rgb(54, 130, 232);
+	--secondary-btn-active-color: rgb(19, 95, 232);
+}
+
+/* Event creation modal */
+
+.modal,
+.event-desc-modal {
+	position: fixed;
+	z-index: 2;
+	top: 25%;
+	left: 50%;
+	background-color: white;
+}
+
+.modal {
+	transform: translate(-50%, 0);
+}
+
+.modal,
+.modal-form,
+.event-desc-modal {
+	border-radius: 10px;
+}
+
+.start-date-text,
+.end-date-text {
+	font-size: 14px;
+}
+
+.modal-form {
+	display: flex;
+	flex-direction: column;
+	width: 500px;
+	padding: 20px 40px 40px;
+	box-shadow: 1px 1px 10px var(--primary-misc-color);
+}
+
+.close-btn {
+	align-self: flex-end;
+	width: 30px;
+	height: 30px;
+	border: none;
+	border-radius: 100%;
+	margin-bottom: 20px;
+	background-color: white;
+}
+
+.event-title {
+	font-size: 22px;
+	margin-bottom: 32px;
+}
+
+.start-date,
+.end-date {
+	color: #555;
+	margin-top: 2px;
+	margin-bottom: 30px;
+}
+
+.description {
+	font-size: 14px;
+	max-width: 420px;
+	resize: none;
+}
+
+:where(.event-title, .start-date, .end-date, .description) {
```

Comment:
kam reikalingas :where cia?

Reply:
Meanwhile there is me learning that ":where" exists :)

What I find in docs is that :where forgives if one of the selectors is invalid. Probably we do not often write invalid CSS selectors so :where would not be required in 99% of cases.